src/librustc_mir/transform/validate.rs

```diff
@@ -244,6 +248,21 @@ impl<'a, 'tcx> Visitor<'tcx> for TypeChecker<'a, 'tcx> {
                             );
                         }
                     }
+                    // See what the deaggregator does with arrays. Until the deaggregator processes
+                    // them, we ignore them here.
```

Comment:
I don't understand this comment. Does this ask the reader to see something, or does this say the code is seeing something?

I think this just say "Arrays are currently not deaggregated and thus can be present in every MIR phase", but I am not sure.